app/controllers/application_controller.rb

```diff
@@ -9,4 +11,11 @@ def check_current_user(message)
     end
   end
 
+  protected
+
+  def record_not_found
+    message = "Item with ID #{params[:id]} not found."
```

Comment:
This also seems to have introduced a bug where I can no longer even view the index page because it redirects me too many times. Seems to be a problem if I have a user_id session param that doesn't exist.